contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -235,6 +286,15 @@ abstract contract PoSValidatorManager is
             validationID == uptimeValidationID, "PoSValidatorManager: invalid uptime validation ID"
         );
 
+        PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+        if (uptime > $._validationUptimeSeconds[validationID]) {
+            $._validationUptimeSeconds[validationID] = uptime;
+        } else {
+            uptime = $._validationUptimeSeconds[validationID];
+        }
+
+        emit UptimeUpdated(validationID, uptime);
```

Comment:
We should maybe only emit this in the `uptime > $._validationUptimeSeconds[validationID]` case?